src/components/app/details/appDetails/AppDetails.tsx

```diff
@@ -633,41 +706,49 @@ export function EventsLogsTabSelector({ onMouseDown = null }) {
             style={{ background: '#2c3354', boxShadow: 'inset 0 -1px 0 0 #0b0f22' }}
             onClick={
                 params.tab
-                    ? () => { }
+                    ? () => {}
                     : (e) => {
-                        history.push(
-                            generatePath(path, { ...params, tab: NodeDetailTabs.MANIFEST }) + location.search,
-                        );
-                    }
+                          history.push(
+                              generatePath(path, { ...params, tab: NodeDetailTabs.MANIFEST }) + location.search,
+                          )
+                      }
             }
         >
             <div className={`pl-20 flex left tab-container ${!!params.tab ? 'dc__cursor--ns-resize ' : 'pointer'}`}>
-                {[NodeDetailTabs.MANIFEST, NodeDetailTabs.EVENTS,
-                ...(kind === Nodes.Pod ? [NodeDetailTabs.LOGS, NodeDetailTabs.TERMINAL] : []),
+                {[
+                    NodeDetailTabs.MANIFEST,
+                    NodeDetailTabs.EVENTS,
+                    ...(kind === Nodes.Pod ? [NodeDetailTabs.LOGS, NodeDetailTabs.TERMINAL] : []),
                 ].map((title, idx) => (
-                    <div key={idx}
-                        className={`tab dc__first-letter-capitalize ${params.tab?.toLowerCase() === title.toLowerCase() ? 'active' : ''}`}
+                    <div
+                        key={idx}
+                        className={`tab dc__first-letter-capitalize ${
+                            params.tab?.toLowerCase() === title.toLowerCase() ? 'active' : ''
```

Comment:
can we change `params.tab?.toLowerCase() === title.toLowerCase()` to `params.tab?.toUpperCase() === title`